components/Weather/index.js

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import Icon from "../Icons";
+
+export default function Weather({ outivity }) {
+  const [weatherData, setWeatherData] = useState(null);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  const fetchData = async () => {
+    try {
+      const response = await axios.get(
```

Comment:
Just for my interest: Can you tell me, why do you use axios?

Reply:
We used axios because it was recommended by the site where we got the API and axios is an alternative to fetch. But now we have rewritten our code and used fetch.